admin/includes/init_includes/init_aidba_install.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+if (!defined('IS_ADMIN_FLAG')) {
+    die('Illegal Access');
+}
+if(!defined('ADDITIONAL_IMAGES_APPROACH')) {
+    $db->Execute("INSERT INTO " . TABLE_CONFIGURATION . " (configuration_title, configuration_key, configuration_value, configuration_description, configuration_group_id, sort_order, set_function, date_added) VALUES ('Additional Images Approach', 'ADDITIONAL_IMAGES_APPROACH', 'old', 'Enable Additional Images Module', '4', '100', 'zen_cfg_select_option(array(\'old\', \'modern\'), ', NOW())");
+}
```

Comment:
Just use INSERT IGNORE INTO, then you don't need the PHP check.  Use short-array syntax for the `zen_cfg_select_option` parameter.

Reply:
aye aye captain. also changing old/modern to legacy/modern